packages/hash/api/src/model/knowledge/orgMembership.model.ts

```diff
@@ -30,11 +31,11 @@ export default class extends EntityModel {
   static fromEntityModel(entityModel: EntityModel): OrgMembershipModel {
```

Comment:
If we're keeping this model class around, shouldn't it now extend `LinkEntityModel` rather than `EntityModel` on line 30?

Reply:
Alfie took care of this in 9fa54ada81cf0e59578d49550d729a9a1dcc90d2 :)